lib/card-react-form-container.js

```diff
@@ -84,17 +85,18 @@
       }
       return React.Children.map(children, (function(_this) {
         return function(child) {
-          var inputsValidationClass, newClassName;
+          var inputAttribute, inputsValidationClass, newClassName;
           if (typeof child !== 'object' || child === null) {
             return child;
           }
-          if (child.props && child.props.name) {
+          inputAttribute = child.props[_this.props.inputAttributeIndentifier];
+          if (child.props && child.props.id) {
```

Comment:
@shatran @TechnicalBro indeed, should be
`if (child.props && inputAttribute) {`
That's a typo I guess ;)